src/main/java/org/sonarlint/intellij/issue/IssueProcessor.java

```diff
@@ -84,11 +98,19 @@ public void process(final SonarLintJobManager.SonarLintJob job, final Collection
     console.info("Found " + issues.size() + end);
   }
 
+  private boolean shouldTrackServerIssues(VirtualFile file, TriggerType trigger, Set<VirtualFile> notAnalyzed) {
+    return trigger == TriggerType.EDITOR_OPEN
+      || trigger == TriggerType.ACTION
+      || !store.getForFile(file).isEmpty() && notAnalyzed.contains(file);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add parentheses to make the operator precedence explicit. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS864)